src/resolvers/hostStats/index.ts

```diff
@@ -0,0 +1,84 @@
+import * as _ from 'lodash';
+import {QueryHostsArgs} from '../../generated/graphql';
+
+import {runQuery} from '../es';
+import {buildFilterQuery} from '../hosts';
+import config from '../../config';
+
+/**
+ * Build query for Elasticsearch based on GraphQL query.
+ */
+function buildESQuery(args: QueryHostsArgs, org_id: string, info: any) {
+
+    const selectionSet = info.fieldNodes[0].selectionSet.selections;
+
+    const dataSelectionSet = _.find(selectionSet, s => s.name.value === 'data');
+    // const sourceList: string = dataSelectionSet.selectionSet.selections.map(
+    //     (o: any) => o.name.value
+    // ) /* .map(translateFilterName) ? */;
+
+    // const query: any = {
+    //     track_total_hits: true,
+    //     _source: sourceList
+    // };
```

Comment:
Commented code blocks should be removed

Reply:
Hi @kruai - appreciate that.  I was leaving the code commented while I worked out what I needed for the actual ElasticSearch query.  If I needed it back, then I hadn't lost it entirely :-)

What I really need is to know whether the ElasticSearch query actually works, and whether it gets the statistics the Dashboard needs - the total number of hosts, and the number of stale and stale warning hosts, meeting the given filter terms.  This should take exactly the same set of filters that the `hosts` query does.  Is the ElasticSearch query working?

Reply:
I haven't been able to get it working locally, no - I was hoping you had tried it out locally as you were coding it. I'll debug locally a bit more and see if it's apparent where the issue is occurring

Reply:
The most info I can get is `Cannot return null for non-nullable field Query.hostStats.`
It makes sense that the field is non-nullable since you want to get something back each time you query it, but I think something in the resolver is returning null.

Reply:
Turns out that the hostStats resolver wasn't in the list of resolvers.